qa/smoke-test-ingest-with-all-dependencies/src/test/java/org/elasticsearch/ingest/IngestDocumentMustacheIT.java

```diff
@@ -31,7 +31,7 @@
 
 import static org.hamcrest.Matchers.equalTo;
 
-public class IngestDocumentMustacheIT extends AbstractMustacheTests {
+public class IngestDocumentMustacheIT extends AbstractMustacheTestCase {
```

Comment:
I don't have an answer for either question. I can certainly track down why this class isn't failing the naming conventions checks though.